src/navigation/TabNavigator.tsx

```diff
@@ -61,7 +62,7 @@ const NavigationRoot = () => {
     >
       <Tab.Screen
         name="Assistant"
-        component={Assistant}
+        component={useExperimentalTripSearch ? Assistant_v2.default : Assistant}
```

Comment:
Så slipper vi spesifisere den her

```suggestion
        component={useExperimentalTripSearch ? Assistant_v2 : Assistant}
```